### PR TITLE
Feat 28 - agreements

### PR DESCRIPTION
**Pull request by vibby**
I left the «Exhibit 1» part rough, as I am not sure if is needed. And I am not very enthoustiastic about doing that boring work without being sure it is useful ;)

### CODE REVIEW

**Comment by m-dotcom** [comment]
@vibby will get back to you today about the «Exhibit 1» part.

**Comment by m-dotcom** [comment]
@vibby Yes, we can add the "Exhibit 1" as a PDF. That should make it easier for you!

Our consultant also notified me about cookies. We have to display a cookie consent banner when EU-based users visit the website for the first time. It is necessary because we use statistics and marketing cookies. No other banners are necessary.

The cookie banner should actually ask our users to consent to our use of statistics and marketing cookies - a mere notice that "We use cookies on Adventist Commons" or "By browsing, you agree to our use of cookies" is not sufficient and does not comply with the GDPR and EU ePrivacy laws. The user has to actually use an affirmative action (e.g., a click, slide, or check) to express consent. Pre-checked boxes are not allowed.

We have to use a GDPR-compliant cookie consent banner.

**Comment by vibby** [comment]
I notice it for pdf. can you send me the pdf as I do not have Microsoft Word ?
I will do that until end of week if you send it.

For GDPR banner, can you open a new ticket please ?

**Comment by m-dotcom** [comment]
@vibby Yes, I will send you the PDF and make a Trello ticket for GDPR banner.

**Comment by m-dotcom** [comment]
@vibby I received update that the entire data processing agreement can be uploaded and liked to as a PDF. I added it to the Trello card where you can download it as a PDF now.

**Comment by m-dotcom** [comment]
@vibby I think we don't need an extra page for the data processing agreement. It is enough to have the ToU and PP pages, and then link from there directly to the PDF.

**Comment by vibby** [comment]
Ok. The link must appear in both ToU and PP ?

**Comment by m-dotcom** [comment]
Yes, check the two documents I uploaded to Trello. If you need it in a different format, let me know.

**Comment by vibby** [comment]
I do not see any link to DPA in docx files PP and ToU. Is that what you are talking about ?
What about the url 
https://adventistcommons.org/data-processing-agreement ? What is must show ?

**Comment by m-dotcom** [comment]
Will highlight them and re-upload them to Trello.

**Comment by vibby** [comment]
ok I got it. Sorry for that so I replace the link here to the real link of the file ? Or better, I make a redirect from https://adventistcommons.org/data-processing-agreement to real file location ?

**Comment by m-dotcom** [comment]
Updated the Trello card. In my opinion just link from the PP and ToU directly to the PDF file. I don't think there is a need to make create a page (https://adventistcommons.org/data-processing-agreement) for it with the PDF.

**Comment by vibby** [comment]
Got it. Done in the afternoon.

File: application/config/routes.php [hunk 1]
```diff
@@ -63,5 +63,8 @@
 $route['account/save'] = 'user/save_account';
 $route['account/save_password'] = 'user/save_password';
 $route['editor/(:num)/(:num)'] = 'editor/index/$1/$2';
+$route['editor/(:num)/(:num)'] = 'editor/index/$1/$2';
 $route['404_override'] = '';
-$route['translate_uri_dashes'] = FALSE;
+$route['terms'] = 'content/termsOfUse';
```
Comment (akjackson1) [inline application/config/routes.php]: For consistency, I'd rather use lowercase function names

File: application/config/routes.php [hunk 2]
```diff
@@ -63,5 +63,8 @@
...
+$route['privacy'] = 'content/privacyPolicy';
```
Comment (akjackson1) [inline application/config/routes.php]: Also, what if we just called the controller "about", then the urls would be like about/privacy-policy, and we wouldn't need to add the custom routes. What do you think? Using "content" as the controller name seems a bit ambiguous

File: application/controllers/Content.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	public function __construct()
+	{
+		parent::__construct();
+		$this->load->library( [ "twig" ] );
+		$this->load->helper( [ "url" ] );
+	}
+	
+	public function termsOfUse()
+	{
+		$this->twig->display( "twigs/content/termsOfUse" );
```
Comment (akjackson1) [inline application/controllers/Content.php]: Lowercase twig file names would be good for consistency also. You can use underscores

**Comment by vibby** [comment]
Done

File: application/config/routes.php (hunk 2 above)
Comment (vibby) [reply application/config/routes.php]: Why not. For urls, they were asked precisely in Trello without the «about», so I just made them that way.
Checked with Michael, it is ok for «/about/»

**Comment by vibby** [comment]
All treated

**Comment by vibby** [comment]
Sorry about that, it was a too much powerful reasch / replace :) I fix it

**Pull request merged**